common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -434,22 +434,6 @@ def setup_multiple_threads(self, thread_count):
         view = MultipleThreadFixture(threads)
         view.push()
 
-    def test_page_scroll_on_thread_change_view(self):
```

Comment:
Removing the "get threads" API call from the non-AJAX view means we're not rendering the DOM elements for the thread list anymore, so the test breaks.

I chatted with Brian about this, and while the current behavior that allows me to do this is not intended/ideal behavior, it's not behavior they will rectify before gutting the whole discussion UI.  When they do that, it's going to be API calls all the way down, so the non-AJAX page load would not be expected to get the thread list anyways.